CHANGELOG.md

```diff
@@ -4,6 +4,7 @@
 * Fixed issue where **update-release-notes** deleted release note file if command was called more than once.
 * Fixed issue where **update-release-notes** added docker image release notes every time the command was called.
 * Fixed an issue where running **update-release-notes** on a pack with newly created integration, had also added a docker image entry in the release notes.
+* Fixed an issue where `XSOAR-linter` did not find *NotImplementedError* outside else clauses in main.
```

Comment:
I think "clauses" are ambiguous maybe a different explanation about what is the change here.